fhir-parent/pom.xml

```diff
@@ -398,7 +398,7 @@
             <dependency>
                 <groupId>org.mockito</groupId>
                 <artifactId>mockito-inline</artifactId>
-                <version>3.7.7</version>
```

Comment:
It'll be interesting to see if we hit any issues

Reply:
Also, may need to update other mockito dependencies

Reply:
yes, thinking we should keep it in sync with mockito-core (just above)...should I update the PR to add that?